ui/app/styles/components/console-ui-panel.scss

```diff
@@ -91,12 +96,22 @@ $console-close-height: 35px;
 .console-ui-alert {
   margin-left: calc(#{$spacing-20} - 0.33rem);
   position: relative;
+  color: var(--token-color-palette-neutral-400);
 
   svg {
     position: absolute;
     left: 0;
     top: 0;
   }
+
+  &.console-ui-alert--error {
+    // HDS tokens are not light enough on the dark background to pass a11y tests.
+    color: lighten(#e52228, 20%);
```

Comment:
can you pass in a token for `#e52228` to lighten? I'm not sure how vars work with the `lighten()` function

Reply:
No, I tried that initially but since it maps to an rgb() value it doesn't work 😢 I'm hoping this is a short-lived problem, but I can add the token this matches to so it comes up in searches